task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -26,7 +26,26 @@ static void Main(string[] args)
                     Console.WriteLine("Frutti");
                 }
                 else
-                {                    
+                {
+                    Console.WriteLine($"Number {i} can't be divided on 2 or 5");
+                }
+            }
+            for (int i = firstInputNumber; i >= secondInputNumber; i--)
```

Comment:
Yeah, your solution will work but your logic is not trivial. It can confuse somebody who will look this code. Why do we need to write 2 loops to check all numbers in interval between 2 numbers? If we input `firstNumber = 1`, `secondNumber = 5` or  `firstNumber = 5`, `secondNumber = 1` in any case we should check interval from 1 to 5. Are you agree?
We can solve this task using only one loop.
We just need to  check all numbers in interval from `min `input number to `max `input number.